docs/docs/Home.md

```diff
@@ -2,11 +2,12 @@
 id: Home
 title: Home
 slug: /
+---
 [![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/LICENSE)
 [![Open Source](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/anitab-org/bridge-in-tech-backend)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/bridge-in-tech-backend/pulse)
 [![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)  
----
+
```

Comment:
Line should be blank. Remove white spaces

Reply:
@devkapilbansal , I'll do that 👍

Reply:
White space is yet to be removed

Reply:
> White space is yet to be removed

Hi @devkapilbansal  , i am not able to trace any white space in my text editor :( can you tell me exactly in which line it is present ?

Reply:
It is not there now